apps/backend/src/excelexport/excelexport.service.ts

```diff
@@ -53,20 +54,60 @@ export class ExcelexportService {
     }
   }
 
+  public generateWorkbook(title: string): void {
+    this.workbook.title = title;
+    this.workbook.created = new Date(format(new Date(), 'yyyy, MM, dd'));
+  }
+
   public addSheet(name: string): void {
     this.workbook.addWorksheet(name);
   }
 
+  /* eslint-disable  */
+  public addHeader(sheetName: string, style: any): void {
```

Comment:
`addHeaderToRow`

Reply:
The header is added to the sheet here ie:
![Screenshot 2023-03-16 at 11 10 29 AM](https://user-images.githubusercontent.com/78890675/225713279-b16bfea1-c197-4b02-a030-101075d5128e.png)

Reply:
resolved?